src/optlang/scipy_interface.py

```diff
@@ -243,15 +243,15 @@ def get_constraint_slack(self, name):
             index = self._get_constraint_index(name)
             return self._slacks[index]
 
-    def optimize(self, method="simplex", verbosity=False, tolerance=1e-9, **kwargs):
+    def optimize(self, method="highs", verbosity=False, tolerance=1e-9, **kwargs):
```

Comment:
Is this a necessary fix or just something you added to the PR because you were making changes?

Reply:
Just to fix warnings. The simplex method is also deprecated and will be removed soon in scipy.